scrunch/helpers.py

```diff
@@ -107,3 +107,14 @@ def _validate_category_rules(categories, rules):
         raise ValueError(
             'Amount of rules should match categories (or categories -1)'
         )
+
+
+def shoji_wrapper(payload, shoji_class='entity'):
+    """
+    receives a dictionary and wraps its content on a body keyed dictionary
+    with the appropiate shoji:<class> attribute
+    """
+    return {
+        'element': 'shoji:{}'.format(shoji_class),
```

Comment:
Note that if you pass `catalog` as value here, the final document will be wrong because it will include a `body`. Where catalogs need to have an `index`.